cmd/mdatagen/main.go

```diff
@@ -99,7 +104,7 @@ func generateFile(tmplFile string, outputFile string, md metadata) error {
 				},
 				"stringsJoin": strings.Join,
 				"inc":         func(i int) int { return i + 1 },
-			}).ParseFS(templateFS, tmplFile))
```

Comment:
If `templateFS` is not being used anymore, I think it should be a full revert.

Reply:
I figured it would be coming back, which is why i left it and the tests here.